nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -1060,7 +1067,7 @@ define(
              *
              * Render the UI according to the FSM
              */
-            function renderUI() {
+            function updateState() {
```

Comment:
renamed this to `updateState`, because it's performing the same action as the bulkImportCell's `updateState` function